### PR TITLE
EDU-105 Update educandu package

### PR DESCRIPTION
**Pull request by ahelmberger**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -29,9 +25,9 @@ const config = {
   emailSenderAddress: processEnv.OMA_EMAIL_SENDER_ADDRESS,
   smtpOptions: processEnv.OMA_SMTP_OPTIONS,
   initialUser: processEnv.OMA_INITIAL_USER ? JSON.parse(processEnv.OMA_INITIAL_USER) : null,
-  exposeErrorDetails: parseBool(processEnv.OMA_EXPOSE_ERROR_DETAILS || false.toString())
+  exposeErrorDetails: parseBool(processEnv.OMA_EXPOSE_ERROR_DETAILS || false.toString()),
+  exportApiKey: processEnv.ELMU_EXPORT_API_KEY,
```
Comment (laurentiu-ilici) [inline src/index.js:29]: This should be OMA_tralala

File: src/index.js [hunk 2]
```diff
@@ -29,9 +25,9 @@ const config = {
...
+  importSources: JSON.parse(processEnv.ELMU_IMPORT_SOURCES || '[]')
```
Comment (laurentiu-ilici) [inline src/index.js:30]: Here as well

**Review by laurentiu-ilici — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (ahelmberger) [reply src/index.js]: Done

File: src/index.js (hunk 2 above)
Comment (ahelmberger) [reply src/index.js]: As well

**Pull request merged**